scripts/build/TestImpactAnalysis/tiaf_driver.py

```diff
@@ -62,4 +62,5 @@ def test_failure_policy(value):
         sys.exit(0)
     except:
         # Non-gating will be removed from this script and handled at the job level in SPEC-7413
-        sys.exit(0)
\ No newline at end of file
+        print("Exception caught by TIAF driver")
```

Comment:
maybe print the exception here so you can get more info if that happens

Reply:
done 8) fixed 👍